zc_common/jwt_auth/authentication.py

```diff
@@ -18,7 +18,7 @@ def __init__(self, **kwargs):
             setattr(self, kwarg, kwargs[kwarg])
 
     def is_authenticated(self):
-        return 'user' in self.roles or 'service' in self.roles
+        return True
```

Comment:
Sorry, not `guest`, the anonymous role. The role assigned to users who did not authenticate. "Unauthenticated" users should not have `is_authenticated()` return True.